client/src/pages/TourPage/TourPage.js

```diff
@@ -19,6 +19,7 @@ import PersonIcon from '@mui/icons-material/Person';
 import { useInView } from 'react-intersection-observer';
 
 import { TourAccordion, TourInfoDialog, TourReasonToChoose } from '../../components';
+import ImageGallery from '../../components/ImageGallery/ImageGallery';
```

Comment:
please check the above line,  we have index.js file for all components, you can add the image gallery there and import it from there

Reply:
fixed